.github/workflows/integration_test.yaml

```diff
@@ -157,10 +157,36 @@ jobs:
       runs-on: ${{ needs.get-runner-image.outputs.runs-on }}
       trivy-image-config: ${{ inputs.trivy-image-config }}
       working-directory: ${{ inputs.working-directory }}
+  build-rocks:
+    name: Build rock
+    uses: ./.github/workflows/build_rocks.yaml
+    needs: get-runner-image
+    with:
+      owner: ${{ github.repository_owner }}
+      registry: ghcr.io
+      runs-on: ${{ needs.get-runner-image.outputs.runs-on }}
+      trivy-image-config: ${{ inputs.trivy-image-config }}
+      working-directory: ${{ inputs.working-directory }}
+  all-images:
+    name: Get rocks or Docker images
+    needs: [build-images, build-rocks]
+    runs-on: ubuntu-latest
+    outputs:
+      images: ${{ env.IMAGES }}
+    steps:
+      - name: Get rocks or fallback to Docker images
+        run: |
+          if [ ${{ needs.build-rocks.outputs.images != '[]' }} ]; then
+            IMAGES='${{ needs.build-rocks.outputs.images }}'
+          else
+            IMAGES='${{ needs.build-images.outputs.images }}'
+          fi
```

Comment:
This means we don't support "mixed" situation where we have dockerfiles and rockcraft files ?
Isn't it more flexible to "merge" both output ?

Reply:
We don't. It is doable but it would be much more complex and we'd need to handle clash scenarios too. I'm not sure it's worth it considering most of the charms will only use one image and rocks will become the default at some point.

Reply:
Well until`rockcraft` isn't mature, we could have a scenario where only a part of the images are available through rocks which requires a mix of rocks and dockerfile
But that's an hypothetical case, and in such case we could decide to keep dockerfile only until the missing support until `rockcraft` supports it.

Reply:
I'd add that support only if we need it, but oc this is my personal opinion